draft-packages/form/KaizenDraft/Form/Primitives/Label/styles.scss

```diff
@@ -60,8 +60,12 @@ $label-start-margin: 10px;
   justify-content: space-between;
   align-items: center;
 
-  .labelText {
+  .prependedLabel {
     @include ca-margin($end: $label-start-margin);
```

Comment:
One small improvement that could be made: `$label-start-margin` is currently set to 10px, we could use `$ca-grid * 0.5` there instead.

Reply:
That's a good point, however, I think `$ca-grid` has been deprecated in favour of `$kz-spacing` tokens and it would probably be better to change over all the form inputs to use those tokens at the same time as a task on its own.

Reply:
I may not be up to date but the last I heard, the `$kz-spacing` tokens were more of an alternative option than a replacement for `$ca-grid`. @ActuallyACat might have a preference on this